### PR TITLE
Added missing keyname variable to generate-keys

### PR DESCRIPTION
**Pull request by capri1989**
Signed-off-by: Kai Wagner <kwagner@suse.com>

### CODE REVIEW

File: generate-keys.sh [hunk 1]
```diff
@@ -21,8 +21,10 @@ if [ -d keys ] ; then
 fi
 mkdir keys
 
+
+KEYNAME="$(grep "keyname:" ./aws.yaml | awk '{ print $2 }')"
```
Comment (smithfarm) [inline generate-keys.sh]: This is an effective way of parsing the yaml in this case, but completely ignores the yaml syntax :-)

**Comment by smithfarm** [comment]
This script is used to generate the SSH keypairs that are used by the individual delegates to access their clusters.

I will need to investigate what the keyname specified in aws.yaml is actually used for.

**Comment by capri1989** [comment]
As I wrote in the documentation I could specify my own "keyname" prefix in the yaml file. But as tested, that's no true as I would use the current user by default and just ignoring what's written in the yaml file?

**Comment by capri1989** [comment]
Oh, is it possible to set different keys for every delegate? I wasn't aware of that. Sure than this solution is not a good idea :D

**Pull request merged**